model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -234,6 +236,60 @@ def replace_config_in_index(mp_cfg: List[int], idx: int, value: int) -> List[int
         updated_cfg[idx] = value
         return updated_cfg
 
+    def _non_configurable_nodes_kpi(self) -> Dict[KPITarget, np.ndarray]:
+        """
+        Computes a KPI vector of all non-configurable nodes in the given graph for each of the KPI target.
+
+        Returns: A mapping between a KPITarget and its non-configurable nodes' KPI vector.
+        """
+
+        non_conf_kpi_dict = {}
+        for target, kpi_value in self.target_kpi.get_kpi_dict().items():
+            if not np.isinf(kpi_value):
+                # Call for the KPI method of the given target - empty quantization configuration list is passed since we
+                # compute for non-configurable nodes
+                if target == KPITarget.BOPS:
+                    kpi_vector = None
+                else:
+                    kpi_vector = self.compute_kpi_functions[target][0]([], self.graph, self.fw_info, self.fw_impl)
```

Comment:
why call a class method with class fields?

Reply:
This is not a class method this is a class variable that contains a tuple of Callable objects (which are methods, but not of this class)